### ISSUE TITLE
Users object in core retaining query.

### ISSUE THREAD

**Issue by Dennor** [Syncano/syncano-node#56]
### Expected behaviour

Each chained call to users.where(/*something*/).doSomething() to be standalone separate query.

### Actual behaviour

`users` property on server object is not an instance of Proxy, unlike `data` property, so each query on Server(ctx).users retains old query.

#### socket.yml
```yaml
name: users-fail-case
description: Description of users-fail-case
version: 0.0.1
endpoints:
  fail-case:
    file: fail.js
    description: abc
    parameters:
```

#### src/fail-case.js

```javascript
import Server from 'syncano-server';

export default async ctx => {
  const {logger, response, users} = Server(ctx);
  const {debug} = logger(ctx.meta.executor);
  let user = await users.updateOrCreate({
    username: 'john_doe',
    password: '|*455|/\\|0|2|)'
  });
  debug(user);
  try {
    await users.delete(user.id);
  } catch (e) {
    debug(e.response);
    return response.json({message: e.message}, 400);
  }
  return response.json({});
};
```

#### package.json
```json
{
  "license": "MIT",
  "dependencies": {
    "syncano-server": "beta"
  },
  "devDependencies": {
    "eslint": "^4.12.1"
  }
}
```

#### npx s version
```
0.53.0
```

Users is not a proxy, so query is retained.

##### updateOrCreate
```json
                { 
                  "username": "john_doe", 
                  "channel_room": null, 
                  "links": { 
                    "self": "/v2/instances/misty-snowflake-6430/users/3/", 
                    "groups": "/v2/instances/misty-snowflake-6430/users/3/groups/", 
                    "reset-key": "/v2/instances/misty-snowflake-6430/users/3/reset_key/" 
                  }, 
                  "user_key": "fb5cc328267b00405bc5c4fd1cac9208fa6f13aa", 
                  "created_at": "2017-12-08T15:50:18.343834Z", 
                  "updated_at": "2017-12-08T15:50:18.343849Z", 
                  "acl": { 
                    "users": { 
                      "3": [ 
                        "read", 
                        "write" 
                      ] 
                    } 
                  }, 
                  "groups": [], 
                  "id": 3, 
                  "channel": null, 
                  "revision": 1 
                } 
```
##### delete
```json
                { 
                  "data": { 
                    "query": "Invalid field name defined: \"password\"." 
                  }, 
                  "body": { 
                    "_readableState": { 
                      "objectMode": false, 
                      "highWaterMark": 16384, 
                      "buffer": { 
                        "head": null, 
                        "tail": null, 
                        "length": 0 
                      }, 
                      "length": 0, 
                      "pipes": null, 
                      "pipesCount": 0, 
                      "flowing": true, 
                      "ended": true, 
                      "endEmitted": true, 
                      "reading": false, 
                      "sync": false, 
                      "needReadable": false, 
                      "emittedReadable": false, 
                      "readableListening": false, 
                      "resumeScheduled": false, 
                      "defaultEncoding": "utf8", 
                      "ranOut": false, 
                      "awaitDrain": 0, 
                      "readingMore": false, 
                      "decoder": null, 
                      "encoding": null 
                    }, 
                    "readable": false, 
                    "domain": null, 
                    "_events": {}, 
                    "_eventsCount": 3, 
                    "_writableState": { 
                      "objectMode": false, 
                      "highWaterMark": 16384, 
                      "needDrain": false, 
                      "ending": true, 
                      "ended": true, 
                      "finished": true, 
                      "decodeStrings": true, 
                      "defaultEncoding": "utf8", 
                      "length": 0, 
                      "writing": false, 
                      "corked": 0, 
                      "sync": false, 
                      "bufferProcessing": false, 
                      "writecb": null, 
                      "writelen": 0, 
                      "bufferedRequest": null, 
                      "lastBufferedRequest": null, 
                      "pendingcb": 0, 
                      "prefinished": true, 
                      "errorEmitted": false, 
                      "bufferedRequestCount": 0, 
                      "corkedRequestsFree": { 
                        "next": null, 
                        "entry": null 
                      } 
                    }, 
                    "writable": false, 
                    "allowHalfOpen": true, 
                    "_transformState": { 
                      "needTransform": false, 
                      "transforming": false, 
                      "writecb": null, 
                      "writechunk": null, 
                      "writeencoding": "buffer" 
                    } 
                  }, 
                  "size": 0, 
                  "timeout": 0, 
                  "url": "https://api.syncano.io/v2/instances/misty-snowflake-6430/users/3/?query=%7B%22username%22%3A%7B%22_eq%22%3A%22john_doe%22%7D%2C%22password%22%3A%7B%22_eq%22%3A%22%7C*455%7C%2F%5C%5C%7C0%7C2%7C)%22%7D%7D&page_size=1", 
                  "status": 400, 
                  "statusText": "Bad Request", 
                  "headers": {} 
                }
```

**Comment by Dennor** [comment]
Made a pull request with fix. If anyone wants to review it and request some changes just mention it.

**Comment by Idered** [comment]
resolved by #58
